root/utility/coordinates.js

```diff
@@ -8,7 +8,15 @@
  */
 
 export function formatCoordinates(coordinates: CoordinatesT) {
-  const {latitude, longitude} = coordinates;
+  // $FlowFixMe
+  if (coordinates === '') {
```

Comment:
As I mentioned somewhere above, we need to fix `build_display_data` so that it returns `undef` instead of an empty string. Then you should remove the `$FlowFixMe` and change the function argument type to `coordinates: ?CoordinatesT` (note the added `?`).

Reply:
Done it